webapp/portlet/src/main/webapp/vue-apps/common/components/RichEditor.vue

```diff
@@ -694,6 +694,7 @@ export default {
           const pattern = new RegExp(`<span class="atwho-inserted" data-atwho-at-query="[^"]*" data-atwho-at-value="${value}"[^>]*>(.*?)</span> </span>`, 'g');
           message = message.replace(pattern, value ? `@${value}` : '');
         });
+      this.inputVal = message;
```

Comment:
can you make this assignment in `updateInput` method only please and rename this method to `replaceSuggesterHtmlToText`

```suggestion
```